ddp-workspace/projects/ddp-sdk/src/lib/models/uploadFile.ts

```diff
@@ -0,0 +1,7 @@
+import { FileUploadResponse } from './fileUploadResponse';
+
+export interface UploadFile extends FileUploadResponse {
```

Comment:
I'm not sure if there is a benefit of putting the local component interfaces into the model folder. This model does not participate in contracts between backend and frontend and is not reused between different parts of the app. My suggetion is keeping such interfaces in the component which uses it. To keep the scope small enough.

Reply:
I don't think it is an issue. 
@MocanaAtBroad, what is your opinion ?

Reply:
I think it is OK to for it to be in the `models` directory as interface gets used from multiple places (component and service) we don't have a better home for it.
But @aglushko-broad observation about a distinction between classes and interfaces that are part of the client/server API vs those that are just for internal consumption is a good one and we should consider doing a little bit of refactoring here. (I did a quick count. We have 63 files directly in model directory, so probably time to reorganize the closet).

Reply:
After last refactor and reviewing latest changes, I am under impression that this interface can just go away. Only thing in it that matters is the File (one attribute) and the upload url is the other argument to actually do the upload and appears it would be easier to just pass it as a function argument. **_MIGHT_** be able to get rid of fileToUpload instance variable altogether, but not 100% sure, but please do take a look. The less instance variables we have, the less state to track and less chance of subtle bugs coming into play.